sas/src/test/resources/python_test.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
```

Comment:
Is this a leftover of testing compatibility with the original Python project? It doesn't seem to be used.